extras/ams_wrapper/src/api/models/model_config.py

```diff
@@ -57,4 +66,33 @@ def validate_type(self, data, **kwargs):
                                   'Invalid config: {}'.format(data))
         if data.get('target_height') and not data.get('target_width'):
             raise ValidationError('target_width must defined if target_height was set. '
-                                  'Invalid config: {}'.format(data))
\ No newline at end of file
+                                  'Invalid config: {}'.format(data))
+
+
+class ModelOutputConfiguration:
+    def __init__(self, output_name: str, value_index_mapping: dict = None,
+                 classes: dict = None, confidence_threshold: float = None,
+                 top_k_results: int = None):
+                 self.output_name = output_name
+                 self.value_index_mapping = value_index_mapping
+                 self.classes = classes
+                 self.confidence_threshold = confidence_threshold
+                 self.top_k_results = top_k_results
```

Comment:
I'm wondering if maybe we could deal with those two parameters as HTTP request parameters so it could be different for each request?

Reply:
I think it would be a good idea to handle these parameters in HTTP request handling, but we are not yet using these config variables, so they can be safely deleted in case when we will implement handling them as HTTP paramters